nowcasting_dataset/config/model.py

```diff
@@ -114,6 +114,13 @@ class Satellite(DataSourceMixin):
     satellite_meters_per_pixel: int = METERS_PER_PIXEL_FIELD
 
 
+class OpticalFlow(DataSourceMixin):
+    """Optical Flow configuration model"""
+
+    previous_timestep_to_use: int = 1
+    final_image_size_pixels: int = IMAGE_SIZE_PIXELS_FIELD
+
+
```

Comment:
do any of the pre saved configs need updating?

Reply:
I'll check, I updated the on_premise.yaml, but I'll check the others too

Reply:
`on_premises.yaml` and `test.yaml` are updated now.

`gcp.yaml` needs updating but TBH we have a bunch of other challenges to get this running on GCP again!  My expectation is that we'll keep the raw data and intermediate Zarrs on `leonardo` and only upload the pre-prepared data to the cloud